### PR TITLE
PHNI-JP cards update prior to 15 Sep

### PR DESCRIPTION
**Pull request by pokeboyexn**

### CODE REVIEW

File: scripts/PHNI-JP/c101203067.lua
Comment (Lyris12) [inline scripts/PHNI-JP/c101203067.lua:1]: already done: #1163

File: scripts/PHNI-JP/c101203069.lua
Comment (Lyris12) [inline scripts/PHNI-JP/c101203069.lua:1]: already done: #1163

File: scripts/PHNI-JP/c101203023.lua
Comment (Lyris12) [inline scripts/PHNI-JP/c101203023.lua:1]: already done: #1163

File: scripts/PHNI-JP/c101203006.lua
Comment (Lyris12) [inline scripts/PHNI-JP/c101203006.lua:1]: already done: #1163

File: scripts/PHNI-JP/c101203007.lua
Comment (Lyris12) [inline scripts/PHNI-JP/c101203007.lua:1]: already done: #1163

**Review by Lyris12 — CHANGES_REQUESTED** [review]

**Comment by pokeboyexn** [comment]
Duplicated scripts removed

**Pull request merged**